ci-operator/config/openshift/kubernetes/OWNERS

```diff
@@ -1,5 +1,38 @@
 # DO NOT EDIT; this file is auto-generated using tools/populate-owners.
-# from https://github.com/openshift/kubernetes/blob/d6035f3e0d79dd05628ef42231beae97806a06ad/OWNERS
+# from https://github.com/openshift/kubernetes/blob/master/OWNERS when master is at 831eec88b8f06e23ab84eb72ecf1c3fce1433446
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
-{}
+filters:
+  .*:
+    approvers:
+    - bgrant0607
+    - brendandburns
+    - dchen1107
+    - jbeda
+    - lavalamp
+    - liggitt
+    - smarterclayton
+    - thockin
+    - wojtek-t
+    reviewers:
+    - brendandburns
+    - dchen1107
+    - jbeda
+    - lavalamp
+    - liggitt
+    - smarterclayton
+    - thockin
+  \.bzl$:
+    approvers:
+    - ixdy
+    reviewers:
+    - ixdy
+  BUILD(\.bazel)?$:
+    approvers:
+    - ixdy
+  go\.(mod|sum)$:
+    labels:
+    - area/dependency
+    required_reviewers:
+    - kubernetes/dep-approvers
```

Comment:
bug?

Reply:
No ... the name is not defined in https://github.com/openshift/kubernetes/blob/master/OWNERS_ALIASES

It is actually a real group.
https://github.com/orgs/kubernetes/teams/dep-approvers

Reply:
Using a github group is wrong in OWNERS file ... sine

[_We use aliases for groups instead of GitHub Teams, because changes to GitHub Teams are not publicly auditable._](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners_aliases)